tutorials/getting_started/another_model_example.md

```diff
@@ -0,0 +1,611 @@
+---
+layout: tutorial
+title: Another Model Example
+id: another-model-example
+---
+
+Our [first tutorial](creating_a_model.md) on building your own model goes through the code you need
+to write to build a named entity tagger.  In this tutorial, we'll do something very similar,
+because, hey, who doesn't like another example?  This time, we'll build a model for classifying
+academic papers, and we'll do it using a [separate
+repository](https://github.com/allenai/allennlp-as-a-library-example), showing how to include
+AllenNLP as a dependency in your project.  We'll step through the commits in that repository,
+detailing all of the steps you need to do in order to get up and running writing your own models.
+
+In the end, there are only about 110 lines of actual implementation code in the `DatasetReader`
+and `Model` classes that we implement.  This tutorial is long because we're explaining what goes
+into those 110 lines, and what AllenNLP does for you behind the scenes so that you only need 110
+lines of code to get a very flexible classifier for academic papers.
+
+## First step: install AllenNLP
+
+In our [first
+commit](https://github.com/allenai/allennlp-as-a-library-example/tree/43b7dac8097a0c4a50c0b25594daa123f3889b75),
+we just add a `requirements.txt` file specifying `allennlp` as a dependency.  We referenced a
+particular commit on github, but you can just give a version number, like `allennlp==0.3`.  Then,
+after creating a python 3.6 environment, you install AllenNLP by running `pip install -r
+requirements.txt`.
```

Comment:
with our current setup don't you also have to install pytorch and the spacy model?

Reply:
Right, thanks.